lib/matplotlib/_mathtext.py

```diff
@@ -2322,6 +2328,24 @@ def _make_space(self, percentage):
         r'\!':         -0.16667,  # -3/18 em = -3 mu
     }
 
+    _unit_multipliers = {
+         # pt|mm|cm|in|ex|em|mu|sp|bp|dd|pc|nc|nd|cc
+        "pt": 1,
+        'mm': 7227/2540,
+        'cm': 7227/254,
+        'in': 7227/100,
+        'ex': 35271/8192,
+        'em': 655361/65536,
+        'mu': 7227/2540000,
```

Comment:
I can't find a ref for mu, nc, nd as standard tex units, perhaps you can link their definition? (as a comment here at least, not in the code)

Reply:
I used some stack exchange thread combined with other sources, but cannot seem to find where I got ex and em from now. I agree with that they shouldn't be there though (just added what was there without really thinking about the meaning).

Here are some better sources:
https://en.wikibooks.org/wiki/LaTeX/Lengths
https://latexref.xyz/Units-of-length.html

(And here is a good discussion why ex and em are bad choices: https://tex.stackexchange.com/questions/4239/which-measurement-units-should-one-use-in-latex
)

Reply:
And here is for mu: https://sv.overleaf.com/learn/latex/Lengths_in_LaTeX